packages/3dom/src/end-to-end-spec.ts

```diff
@@ -95,11 +106,42 @@ suite('end-to-end', () => {
     expect(messageEvent.data).to.be.equal(material.name);
   });
 
+  suite('configuring textures', () => {
+    test('can change the base color texture image', async () => {
+      const {executionContext, graft, gltf} = await prepareConstructsFor(
+          ASTRONAUT_GLB_PATH, ['messaging', 'material-properties', 'textures']);
+      const textureURL = imageURL(64, 64);
+
+      executionContext.changeModel(graft);
+
+      executionContext.eval(`
+self.addEventListener('message', function(event) {
+  var textureURL = event.data;
+  model.materials[0].pbrMetallicRoughness.baseColorTexture.texture.source.setURI(textureURL);
+});
+
+self.postMessage('ready');
```

Comment:
What does this do? It doesn't seem anyone is listening for a 'ready' message.

Reply:
`ready` is the `event.data` of the `message` event we are listening for in the worker.